src/game/Spell.cpp View file @ 35ade7f
@@ ATTENTION CASE 31306 NORMALEMENT EXISTANT SUR TGC , LE SUPPRIMER (CAR USELESS) AVANT D'APPLIQUER CE FIXE (QUI LUI MARCHE) @@
void Spell::SendCastResult(SpellCastResult result)
@@ -6280,6 +6289,10 @@ bool Spell::CheckTarget(Unit* target, SpellEffectIndex eff)
      if(target->GetTypeId() == TYPEID_PLAYER && target == m_caster)
        return false;
      break;
+   case 31306:                      // Anetheron Carrion Swarm (1.22 = 70ï¿½) @Kordbc
+     if(!m_caster->HasInArc(1.22f, target))
+       return false;
+     break;
    default: break;
    }